vendor/github.com/rancher/steve/pkg/accesscontrol/policy_rule_index.go

```diff
@@ -153,9 +153,7 @@ func (p *policyRuleIndex) getClusterRoleBindings(subjectName string) []*rbacv1.C
 	if err != nil {
 		return nil
 	}
-	sort.Slice(result, func(i, j int) bool {
-		return result[i].Name < result[j].Name
-	})
+	// call chains GetByIndex -> ByIndex -> IndexKeys (already sorted by name )
```

Comment:
How can we ensure the the result sorted? Can you reference the code here?

Reply:
We didn't sort the result in the wrangler-api implement, so we still need sort here.

https://github.com/aiwantaozi/prometheus-auth/blob/monitor-refactor/vendor/github.com/rancher/wrangler-api/pkg/generated/controllers/rbac/v1/clusterrolebinding.go#L246-L256

https://github.com/aiwantaozi/prometheus-auth/blob/monitor-refactor/vendor/k8s.io/client-go/tools/cache/thread_safe_store.go#L180-L198